src/client/components/Competitors.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from "react";
+
+import "./styling/competitors.css";
+
+export const Competitors = () => {
+  const [newCompetitionName, setNewCompetitionName] = useState("");
+  const [competitors, setCompetitors] = useState([]);
+  const [newCompetitorName, setNewCompetitorName] = useState("");
```

Comment:
I don't mind this so much (so I won't let it stop me approving), but I reckon I'd prefer a single competitor object with a number of properties ;) That way, you'd only need a single `setCompetitor`, which you'd set with something like `setCompetitor({...competitor, [key]: value})`